virttest/qemu_devices/qcontainer.py

```diff
@@ -1721,6 +1722,7 @@ def images_define_by_variables(self, name, filename, pci_bus, index=None,
         :param image_encryption: ImageEncryption object for image
         :param image_access: The logical image access information object
         :param external_data_file: external data file for qcow2 image
+        :param copy_on_read: if support copy-on-read filter
```

Comment:
copy_on_read -> image_copy_on_read

Reply:
Add a brief introduction to image_copy_on_read to virttest/shared/cfg/base.cfg, you can refer to image_throttle_group, in your case, image_copy_on_read is a string, refer to utils_params.get_boolean, 'yes'/'on'/'true' can be treated as True, while 'no'/'off'/'false' is False

Reply:
Have updated, thanks zhenchao.